### DISCUSSION TITLE
Can't download from loom.com

### DISCUSSION THREAD

**Discussion by tbll** [aclap-dev/video-downloadhelper#7] [Help and Q&A]
### Your question

I get an error that says: `Exit code: 1 `.

### Addon & CoApp version and details

version: 8.2.0.2
channel: stable
build date: 2024-01-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.12",
  "binary": "/Users/paul/git/vdhcoapp/dist/mac/arm64/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "arm64",
    "node": "18"
  },
  "home": "/Users/paul",
  "converterBinary": "/Users/paul/git/vdhcoapp/dist/mac/arm64/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.1"
}

**Comment by paulrouget** [comment]
Right. We are aware of this issue. We know what's going on, and that's in our TODO list! Thanks for reporting.

**Comment by ElMeik** [comment]
Team do we have an update since I caln download as well any embedded loom videos?

**Comment by mosesjaxon** [comment]
@paulrouget, was this updated in the 9.1 release?

**Reply by jcfrog** [reply]
Sorry there was an error in the update page, it's actually a 9.3 version.
It was a fix for Youtube videos detection issue.

**Comment by guice** [comment]
I'm +1 myself to this. Running into this today trying to get a private video associated to a call I had earlier.

**Comment by dbrownxx** [comment]
Same - would love to be able to download Loom!
